src/main/service/CachingService.ts

```diff
@@ -51,13 +51,12 @@ export class CachingService {
    * @param placeId - place ID of the restaurant
    */
   async getLatLonValue(placeId: string): Promise<CachingObject> {
-    return new Promise((resolve, reject) => {
+    return new Promise((resolve) => {
       this.client.get(placeId, (err: Error | null, data: string | null) => {
-        if (err) {
-          // todo: https://github.com/ubclaunchpad/foodies/issues/89
-          reject(err);
+        if (err || !data) {
+          return resolve(JSON.parse('{}'));
```

Comment:
Wait how come this is resolved? 

If there is no more lat lon in the cache for a specific placeId, we would never get the lat lon again unless we call `POST /promotions`